concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/contractexample/parameters/BalanceOfQuery.java

```diff
@@ -0,0 +1,19 @@
+package com.concordium.sdk.examples.contractexample.parameters;
+
+import com.concordium.sdk.types.AbstractAddress;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * Represents a '<a href = "https://github.com/Concordium/concordium-rust-smart-contracts/blob/main/concordium-cis2/src/lib.rs#L1138">BalanceOfQuery</a>' used in different smart contracts
+ */
+@Getter
+@AllArgsConstructor
+public class BalanceOfQuery<T extends TokenId> {
+    @JsonProperty("token_id")
+    private T tokenId;
```

Comment:
Document fields please. 

Imagine a developer sees this and he has 0 experience with the SDK. 

For example, "What is an `AbstractAddress`". Well, it's either an address of a contract or an account.